cmp/grid/columns/Column.js

```diff
@@ -610,7 +610,16 @@ export class Column {
                 throw XH.exception('Column editor must be a HoistComponent or a render function');
             });
             ret.cellClassRules = {
```

Comment:
I think ret.cellClassRules and agOptions.cellClassRules need to specially merged so as to really be combined together.  As is now, an app's agOptions.cellClassRules will wipe out ret.cellClassRules won't it?

I have a client app with 12 instances of cellClassRules...

Reply:
Yes, agree 100% and that's something that I want to handle very soon via longstanding ticket #1331.

This PR doesn't change the current (suboptimal) behavior, which was introduced with the addition of the validation display to the first inline editing release.

Let me see if I can bump up that other ticket and get a clean approach into a dedicated PR for this overall Hoist release. I don't think it should hold up this merge, but agree it's a priority.

Reply:
ok, sounds good.